https://issues.redhat.com/browse/ACM-12788 (#6697)
open-cluster-management/rhacm-docs: release_notes/known_issues_observability.adoc

//2.11:ACM-11407

Custom Observatorium API and Alertmanager URLs only support intermediate components with TLS passthrough. If both custom URLs are pointing to the same intermediate component, you must use separate sub-domains because {ocp-short} routers do not support two separate route objects with the same host.

[#metrics-displayed-obs-ocp311]

> == Metrics are only displayed from {ocp-short} 3.11 managed clusters 

Author: ```suggestion
== Metrics from {ocp-short} 3.11 managed clusters are not displayed from the hub cluster
```

//2.11:ACM-12773

> If a managed hub cluster with Observability enabled imports an {ocp-short} 3.11 cluster (deprecated) as managed cluster, the metrics are only displayed from the managed cluster.

Reviewer: According to the information in the JIRA, the issue at hand is that metrics from managed clusters running OCP 3.11 won't be collected by ACM Observability and thus they won't be visible from the Hub in any way. 

They will still be collected by the Cluster Monitoring Operator that runs inside the OCP cluster.

> If a managed hub cluster with Observability enabled imports an {ocp-short} 3.11 cluster (deprecated) as managed cluster, the metrics are only displayed from the managed cluster.

Author: ```suggestion
Metrics from an {ocp-short} 3.11 managed cluster (deprecated) are not displayed from the hub cluster because the metrics are not collected from your {product-title-short} hub cluster.
```

> //are users able to make updates to change this outcome?

Reviewer: We expect a fix for this to be released in 2.11.1. There's no action required from the customer side besides doing the upgrade when it is available.

> //are users able to make updates to change this outcome?

Author: ```suggestion
```
